pbt-gen/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "pbt-gen"
+version = "0.1.0"
+authors = ["Shon Feder <shon@informal.systems>"]
+edition = "2018"
+description = """
+            An internal crate providing proptest generators used across our
+            crates and not depending on any code internal to those crates.
+            """
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+chrono = { version = "0.4", features = ["serde"] }
```

Comment:
Can we feature-guard the `chrono` dev and associated proptest strategies? If this crate grows or end up being published, it would be nice to only pull in whatever strategies (and their dependencies) are actually needed by the downstream project.

Reply:
I guess at the moment there is only a single set of strategies, so it does not matter much. But it could still be feature-guarded and enabled by default.

Reply:
Done!